docs/DeveloperGuide.adoc

```diff
@@ -354,31 +364,52 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Pros: Easy to understand for new Computer Science student undergraduates to understand, who are likely to be * the new incoming developers of our project.
 ** Cons: Logic is duplicated twice. For example, when a new command is executed, we must remember to update * both `HistoryManager` and `UndoRedoStack`.
 * **Alternative 2:** Use `HistoryManager` for undo/redo
-** Pros: We do not need to maintain a separate stack, and just reuse what is already in the codebase.
-** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
+** Pros: We do not need to maintain a separate stack, and can just reuse what is already in the codebase.
+** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Furthermore, the Single Responsibility Principle and Separation of Concerns are violated as `HistoryManager` now needs to do two * different things.
 // end::undoredo[]
 
 // tag::dataencryption[]
 === [Proposed] Data Encryption
+==== Rationale
+Data encryption is key to any App that deals with personal data of individuals.
```

Comment:
Maybe change the word "key" to something like "important for" to avoid confustion :P

Reply:
that was meant to be a pun 😛

Reply:
Lol in that case, it's up to you 😅